### DISCUSSION TITLE
Stellarium Focal Lenght problem

### DISCUSSION THREAD

**Discussion by Prudentis** [Stellarium/stellarium#1672] [Q&A]
Whenever I try to set a focal length over 999 stellarium automatically adds two zeroes, so my 2350 becomes 235000.
Adding a decimal point doesn't help either. What am I doing wrong?

**Comment by alex-w** [comment]
Please show screenshot

**Comment by Prudentis** [comment]
I don't think a screenshots adds any information but here you go:

![image](https://user-images.githubusercontent.com/84249275/118395633-aad7bd80-b64b-11eb-9786-32f30ba50a31.png)

I type in 2350 and as soon as I exit the FL box, Stellarium immediatenly adds 2 zeros without a decimal point. In fact Stellarium adds two zeroes every time I add any number > 999. When I enter a number <= 999 it adds 2 zeroes and a decimal point, which is, what I would expect.
When I enter 2350.00, the decimal point vanishes and I end up with 235000 again.

![image](https://user-images.githubusercontent.com/84249275/118395681-f25e4980-b64b-11eb-8632-86bdf2fbd1f0.png)

**Reply by alex-w** [reply]
I cannot reproduce the issue

**Comment by Prudentis** [comment]
Well, that's what I thought, since I couldn't find the problem discussed anywhere on the web. I only posted the question here, since I have not found a solution anywhere.
So what happens in your Stellarium when you enter let's say 2500? Does Stellarium add .00 or does it stay at 2500?
It is still a real problem that occurs in my Stellarium and I can't add any of my bigger scopes.
Maybe someone has an idea, what could cause the issue... I mean, I am not imagining things :D
Could German localization be a problem? Where does Stellarium get the decimal delimiter from?

**Comment by Prudentis** [comment]
EDIT: I THINK Stellarium adds the two zeros but actually simultaneously changes the scale from 10^−3 m to 10^−5 m, since the sensor size shown is the same as if I put in 999mm and add a 2.35 barlow

**Reply by gzotti** [reply]
I cannot reproduce this in Stellarium version 0.20.4 or 0.21.0 on Win10/German either. Do you run any other but the latest verison?
